arm/helpers/helpers.go

```diff
@@ -0,0 +1,47 @@
+package helpers
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	"github.com/Azure/azure-sdk-for-go/arm/resources/resources"
+)
+
+// BuildDeployment is a helper method that create a deployment object
```

Comment:
I think this needs to be beefed up (why would somebody want to call this method) and include links about ARM templates.  Also describe the return params, e.g. what do I actually do with the returned deployment object?

Reply:
Fixed